client/galaxy/scripts/onload.js

```diff
@@ -19,6 +19,7 @@ var POPUPMENU = require( 'ui/popupmenu' );
 window.make_popupmenu = POPUPMENU.make_popupmenu;
 window.make_popup_menus = POPUPMENU.make_popup_menus;
 window.init_tag_click_function = require( 'ui/autocom_tagging' );
+window.Toastr = require( 'libs/toastr' );
```

Comment:
I mentioned this in IRC, but changes like this make me uneasy, similar to the concerns addressed by https://github.com/galaxyproject/galaxy/pull/3731.  Webhooks are intended to be an optional feature, but this change loads the Toastr library on every single page regardless of webhooks being enabled.  The toastr library should be imported when necessary from the webhook itself.